frontend/package.json

```diff
@@ -4,91 +4,91 @@
   "license": "BSD-2-Clause",
   "private": false,
   "dependencies": {
-    "@formatjs/intl-locale": "^3.0.11",
-    "@formatjs/intl-pluralrules": "^5.1.8",
-    "@formatjs/intl-relativetimeformat": "^11.1.8",
-    "@formatjs/macro": "^0.2.8",
+    "@formatjs/intl-locale": "^3.1.1",
+    "@formatjs/intl-pluralrules": "^5.1.10",
+    "@formatjs/intl-relativetimeformat": "^11.1.10",
     "@hotosm/id": "^2.21.1",
     "@hotosm/iso-countries-languages": "^1.1.2",
-    "@lokibai/react-use-copy-clipboard": "^1.0.4",
+    "@lokibai/react-use-copy-clipboard": "^1.0.5",
     "@mapbox/mapbox-gl-draw": "^1.3.0",
     "@mapbox/mapbox-gl-geocoder": "^5.0.1",
-    "@mapbox/mapbox-gl-language": "^0.10.1",
+    "@mapbox/mapbox-gl-language": "^1.0.1",
     "@placemarkio/geo-viewport": "^1.0.0",
-    "@sentry/react": "^7.31.1",
-    "@sentry/tracing": "^7.31.1",
-    "@tmcw/togeojson": "^4.7.0",
+    "@sentry/react": "^7.48.0",
+    "@sentry/tracing": "^7.48.0",
+    "@tmcw/togeojson": "^5.6.0",
     "@turf/area": "^6.5.0",
     "@turf/bbox": "^6.5.0",
     "@turf/bbox-polygon": "^6.5.0",
+    "@turf/centroid": "^6.5.0",
     "@turf/helpers": "^6.5.0",
     "@turf/intersect": "^6.5.0",
     "@turf/line-to-polygon": "^6.5.0",
     "@turf/transform-scale": "^6.5.0",
     "@turf/truncate": "^6.5.0",
-    "@uiw/react-md-editor": "^3.20.2",
-    "RapiD": "facebookincubator/rapid#rapid-v1.1.9-tm.1",
-    "axios": "^0.27.2",
-    "chart.js": "^3.7.1",
-    "chartjs-adapter-date-fns": "^2.0.0",
+    "@uiw/react-md-editor": "^3.20.8",
+    "RapiD": "facebook/RapiD#rapid-v1.1.9-tm.1",
+    "axios": "^1.3.5",
+    "chart.js": "^4.2.1",
+    "chartjs-adapter-date-fns": "^3.0.0",
     "date-fns": "^2.29.3",
-    "dompurify": "^2.4.3",
-    "downshift-hooks": "^0.8.1",
-    "final-form": "^4.20.8",
+    "dompurify": "^3.0.2",
+    "final-form": "^4.20.9",
     "fromentries": "^1.3.2",
     "humanize-duration": "^3.28.0",
     "mapbox-gl": "^1.13.2",
     "mapbox-gl-draw-rectangle-mode": "^1.0.4",
-    "marked": "^4.2.5",
+    "marked": "^4.2.12",
     "osmtogeojson": "^3.0.0-beta.5",
-    "query-string": "^7.1.3",
-    "react": "^17.0.2",
-    "react-accessible-accordion": "^3.3.4",
+    "prop-types": "^15.8.1",
+    "query-string": "^8.1.0",
+    "react": "^18.2.0",
+    "react-accessible-accordion": "^5.0.0",
     "react-calendar-heatmap": "^1.9.0",
-    "react-chartjs-2": "^4.1.0",
-    "react-datepicker": "^4.8.0",
-    "react-dom": "^17.0.2",
-    "react-dropzone": "^11.3.4",
+    "react-chartjs-2": "^5.2.0",
+    "react-click-outside": "^3.0.1",
+    "react-datepicker": "^4.10.0",
+    "react-dom": "^18.2.0",
+    "react-dropzone": "^14.2.3",
```

Comment:
A file dialog opens consistently whenever you switch between the Write and Preview tabs on the markdown editor.

Reply:
I haven't seen that. I'll see if I can reproduce.

Reply:
I wasn't able to reproduce.

Was this _any_ markdown editor, or a specific markdown editor?

Reply:
Like I said, I haven't seen this. I'll push what I currently have locally so that we are looking at the same source code.

Reply:
I discovered that the problem only happens in Chrome and not in Firefox. Also, when you press anywhere on the comment box container, the file dialog opens consistently. It's extremely strange.

Reply:
I've been investigating the issue further, and the problem seems related to the `react-dropzone` bump. I don't know if this could be related to [Using open() on Click](https://github.com/react-dropzone/react-dropzone#using-open-on-click) section on their docs, but adding `noClick: true` on the `DROPZONE_SETTINGS` did prevent the unexpected behavior on my end.